src/index.ts

```diff
@@ -38,16 +38,16 @@ class ErrorWithCode extends Error {
 }
 
 export class GoogleToken {
-  token: string|null;
-  expiresAt: number|null;
+  token: string|null = null;
```

Comment:
There were some places throughout the original code that would assign `null` to properties rather than `undefined`.  I agree that this isn't ideal, but changing that is a potential semver major breaking change :(   

As a general rule, I really wonder if we shouldn't avoid null all together.